scripts/boxplot_T_star.m

```diff
@@ -0,0 +1,24 @@
+clear all
```

Comment:
Can't these boxplot_ scripts be made more flexible? Why separate scripts to make Boxplots from different data types?